src/ToolBox/SOS/lldbplugin/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+project(sosplugin)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+set(LLVM_DIR "/usr/lib/llvm-3.5")
```

Comment:
We need to make this conditional for Linux only and (possibly) fall back to reading path to LLDB headers/libs from environment variables on OS X.